server/server.go

```diff
@@ -142,6 +143,9 @@ func Start() {
 func indexHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	w.Write([]byte("This is the matterbuild server."))
 }
+func healthHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	w.Write([]byte("Healthy!"))
```

Comment:
It might be worth to add additional information like the commit sha to the response to make debugging easier. https://github.com/mattermost/mattermost-marketplace/blob/master/internal/api/health_check.go is how the Marketplace does implement a `health` endpoint.

Reply:
I was about to do that on separate PR after this gets merged to put extra information, such as commit SHA, build time, build OS/ARCH, etc , to the final artifact, and didn't want to bloat this PR.